src/main.rs

```diff
@@ -217,9 +216,14 @@ impl App {
     }
 
     fn programming_language_from_args(&self, args: &ArgMatches) -> Result<String> {
-        match args.get_one::<String>("PROGRAMMING_LANGUAGE") {
-            Some(s) => Ok(s.to_owned()),
-            None => Err(anyhow!("No programming language given")),
+        let lang = args
+            .get_one::<String>("PROGRAMMING_LANGUAGE")
+            .context("Should have a programming language")?
+            .as_str();
+
+        match lang {
+            "ruby" => Ok(String::from("ruby")),
```

Comment:
This should be dynamically generated somehow, and it should also probably support aliases such as `rb` for `ruby`.

Reply:
Yeah atm this is just there to guide (myself) through the `generate-stub` command.